modules/core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/SchedulerServiceImpl.java

```diff
@@ -102,7 +113,9 @@ protected void setupScheduler() throws IOException, SchedulerException {
             schedulerFactory.initialize(props);
         }
         scheduler = schedulerFactory.getScheduler();
-        scheduler.start();
+        // delay Quartz scheduler start to avoid unique key constraints violation with qrtz_LOCKS table
+        ConfigurationService cs = Framework.getService(ConfigurationService.class);
+        scheduler.startDelayed(cs.getInteger(CLUSTER_START_DELAY_PROP, CLUSTER_START_DELAY_DEFAULT));
```

Comment:
I'd prefer if the value and default was expressed as "5s" and that you call `ConfigurationService.getDuration`. Then convert the `Duration` to seconds when passing it to `startDelayed` if that's what this method requires.

Reply:
I did this change to use Duration